Cargo.toml

```diff
@@ -8,12 +8,16 @@ readme = "README.md"
 description = "multibase in rust"
 homepage = "https://github.com/multiformats/rust-multibase"
 repository = "https://github.com/multiformats/rust-multibase"
-keywords = ["ipld", "ipfs", "multihash", "multibase", "cid"]
+keywords = ["ipld", "ipfs", "multihash", "multibase", "cid", "no_std"]
+
+[features]
+default = ["std"]
+std = ["data-encoding/std"]
 
 [dependencies]
-base-x = "0.2"
-data-encoding = "2.2"
-data-encoding-macro = "0.1.8"
+base-x = { version = "0.2.6", default-features = false }
```

Comment:
I think this should be 0.2.7.

Reply:
ah, clicked an old docs.rs link... curious that it built without complaint when I tested it...